setup.cfg

```diff
@@ -0,0 +1,22 @@
+[metadata]
```

Comment:
This file is one of the vital ingredients in making an installable python package.  Here, you say what version the package will have.  before you release the package, you need to update the version here.

Reply:
I will keep a note on this. Thanks!